src/vs/workbench/parts/execution/electron-browser/terminal.contribution.ts

```diff
@@ -99,7 +112,18 @@ export class OpenConsoleAction extends Action {
 			}
 		}
 
-		this.terminalService.openTerminal(pathToOpen);
+		const configuration = this.configurationService.getConfiguration<ITerminalConfiguration>();
```

Comment:
I think we actually want to make this decision in `ExplorerViewerActionContributor`, `OpenConsoleAction` is an action on the external/native terminal right now. Currently having `terminal.terminaKind` as `"integrated"` would make ctrl+shift+c behave similar to ctrl+` by creating and showing a new terminal - we want both to be able to be used at the same time.